### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by DannyRuchtie**
First version of CodeSandbox Changelog. 
Added changelog blog post: 
/changelog/

- How do we want to handle version name? 
- How far do we want to go back in time? 
(I propose to have a post for everything up until v3 and have one big posts only for: v2.5, v2 and v1) 

Todo:
- [ ] Add more content
- [ ] Add menu item 

Screenshot 2020-04-05 at 22 33 26
Screenshot 2020-04-05 at 22 33 22

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3843/commits/d8ebc4df0d6d599febf4856c746d87fb818823a9 is at https://pr3843.build.csb.dev/s/new.

File: packages/homepage/content/changelog/new-look-editor/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<ul>
+  <li>Significant stability and performance improvements</li>
+  <li>When the Slack integration access has been revoked or turned off we send out a notification</li>
+  <li>Fix to issue URLs containing backtick characters</li>
+  <li>GitHub autolinking documentation improvements</li>
+  <li>Project team links and issue peek now work better on mobile devices</li>
+  <li>Removed team deletion from the command menu. To remove a team, visit the team settings page</li>
+<li>API: Fixed pagination on <code>users</code> query</li>
+</ul>
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/new-look-editor/index.md:29]:
```suggestion
- Significant stability and performance improvements
- When the Slack integration access has been revoked or turned off we send out a notification
- Fix to issue URLs containing backtick characters
- GitHub autolinking documentation improvements
- Project team links and issue peek now work better on mobile devices
- Removed team deletion from the command menu. To remove a team, visit the team settings page
- API: Fixed pagination on <code>users</code> query
```

File: packages/homepage/content/changelog/redefining-code-collaboration/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+reduced friction in sharing ownership of sandboxes within a group.
+
+#### Features
+
+<ol>
+  <li>Significant stability and performance improvements</li>
+  <li>Project team links and issue peek now work better on mobile devices</li>
+  <li>Removed team deletion from the command menu. To remove a team, visit the team settings page</li>
+  <li>API: Fixed pagination on  query</li>
+</ol>
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/redefining-code-collaboration/index.md:26]:
```suggestion
1. Significant stability and performance improvements
2. Project team links and issue peek now work better on mobile devices
3. Removed team deletion from the command menu. To remove a team, visit the team settings page
4. API: Fixed pagination on query
```

File: packages/homepage/content/changelog/redefining-code-collaboration/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+<ul>
+  <li>Significant stability and performance improvements</li>
+  <li>When the Slack integration access has been revoked or turned off we send out a notification</li>
+  <li>Fix to issue URLs containing backtick characters</li>
+  <li>GitHub autolinking documentation improvements</li>
+  <li>Project team links and issue peek now work better on mobile devices</li>
+  <li>Removed team deletion from the command menu. To remove a team, visit the team settings page</li>
+<li>API: Fixed pagination on query</li>
+</ul>
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/redefining-code-collaboration/index.md:38]:
```suggestion
- Significant stability and performance improvements
- When the Slack integration access has been revoked or turned off we send out a notification
- Fix to issue URLs containing backtick characters
- GitHub autolinking documentation improvements
- Project team links and issue peek now work better on mobile devices
- Removed team deletion from the command menu. To remove a team, visit the team settings page
- API: Fixed pagination on query
```

File: packages/homepage/content/docs/2-live.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
 ---
-title: Collaboration with Live
+title: v3.6
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:2]: You changed a docs page, so not sure if this was intended...? 🤔

**Comment by MichaelDeBoey** [comment]
We once introduced a changelog (see #383), but it's been removed (see #1553).

We also have blogposts for [v1.5](https://pr3843.build.csb.dev/post/codesandbox-1-5-changelog), [v2.0](https://pr3843.build.csb.dev/post/announcing-codesandbox-2-0), [v2.5](https://pr3843.build.csb.dev/post/announcing-codesandbox-2-5) & [v3](https://pr3843.build.csb.dev/post/announcing-codesandbox-v3), so maybe we should do v3 until v3.6 in detail and have a changelog for v1.5, v2.0 & v2.5 since we know what changed there...? 🤔

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/Code 2.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/Code 2.tsx:1]: Is this file needed in this change?

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/Markdown 2.tsx [hunk 6]
```diff
@@ -0,0 +1,174 @@
+import React, { useState } from 'react';
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/Markdown 2.tsx:1]: Is this file needed in this change?

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/Code 2.tsx (hunk 5 above)
Comment (DannyRuchtie) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/Code 2.tsx:1]: Not sure where this came from

**Comment by DannyRuchtie** [comment]
@MichaelDeBoey 
Thx for the feedback. Ives was talking about wanting to bring back the change-log. So I decided to make it a weekend project. All document pages are examples, I wanted to have a running environment so I can discuss it with Ives and Gareth ( a PR probably is not the best way to do this, but it was the only way I could think of :) )

File: packages/homepage/content/docs/2-live.md (hunk 4 above)
Comment (DannyRuchtie) [reply packages/homepage/content/docs/2-live.md:2]: Whoops your right! This should not have been changed.

**Comment by MichaelDeBoey** [comment]
@DannyRuchtie Just had a look and it looks really slick imo!
Great job for a non-coder 😉

File: packages/homepage/content/changelog/3-6-1/index.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+</ol>
+
+#### Fixes
+
+<ul>
+  <li>The template icon popup now closes upon selecting the icon</li>
+  <li>Fixed an alignment issue with the Share icon</li>
+  <li>We now correctly reset live counters after a sandbox is changed</li>
+  <li>Fixed up an issue impacting live throttling when editing a sandbox solo</li>
+</ul>
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-1/index.md:25]:
```suggestion
- The template icon popup now closes upon selecting the icon
- Fixed an alignment issue with the Share icon
- We now correctly reset live counters after a sandbox is changed
- Fixed up an issue impacting live throttling when editing a sandbox solo
```

File: packages/homepage/content/docs/2-live.md (hunk 4 above)
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:2]:
```suggestion
title: Collaboration with Live
```

File: packages/homepage/content/docs/2-live.md [hunk 8]
```diff
@@ -1,10 +1,12 @@
...
 authors: ['CompuIves']
 description:
   With CodeSandbox Live you can share your sandbox for real time collaboration.
 ---
 
+# Collaboration with Live
+
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:9]:
```suggestion
```

File: packages/homepage/content/changelog/3-4-9-small-fixes/index.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+###
+
+#### Fixes
+
+- Improved performance when switching between active files
+- We now explicitly set Same-Site:None on preview secret cookies ahead of
+  changes to the way they'll be handled by Chrome in the future
+- We fixed an issue with deleting templates
+- Thanks to [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/3695), you can no longer add blank font and resource names
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-4-9-small-fixes/index.md:17]:
```suggestion
- Thanks to [@jyash97](https://github.com/jyash97), you can no longer add blank
  font and resource names
  ([#3695](https://github.com/codesandbox/codesandbox-client/pull/3695))
```

File: packages/homepage/content/changelog/3-6-1/index.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+slug: bug fixes v3.6.1
+date: 2020-04-01
+description: This release includes bug fixes.
+---
+
+###
+
+#### Features
+
+1. [@tanmoyopenroot](https://github.com/codesandbox/codesandbox-client/pull/3738) added a keyboard shortcut Ctrl + Backtick to toggle opening and closing the console
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-1/index.md:13]:
```suggestion
1. [@tanmoyopenroot](https://github.com/tanmoyopenroot) added a keyboard
   shortcut Ctrl + Backtick to toggle opening and closing the console
   ([#3738](https://github.com/codesandbox/codesandbox-client/pull/3738))
```

File: packages/homepage/content/changelog/3-6-1/index.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+2. Thanks to [@jamime](https://github.com/codesandbox/codesandbox-client/pull/3730), sandboxes can now consume packages that use an npm alias
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-1/index.md:14]:
```suggestion
2. Thanks to [@jamime](https://github.com/jamime), sandboxes can now consume
   packages that use an npm alias
   ([#3730](https://github.com/codesandbox/codesandbox-client/pull/3730))
```

File: packages/homepage/content/changelog/3-6-1/index.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+#### Fixes
+
+- Fixed an alignment issue with the Share icon
+- [@tanmoyopenroot](https://github.com/codesandbox/codesandbox-client/pull/3759) made the template icon popup close upon selecting the icon
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-1/index.md:19]:
```suggestion
- [@tanmoyopenroot](https://github.com/tanmoyopenroot) made the template icon
  popup close upon selecting the icon
  ([#3759](https://github.com/codesandbox/codesandbox-client/pull/3759))
```

File: packages/homepage/content/changelog/3-0-0/index.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+date: 2019-03-19
+description: Redesigned editor and VS Code extensions support
+---
+
+### 
+
+#### Features
+
+1. VS Code is now the core editor in CodeSandbox, enabling keybindings, user snippets, commands, 
+editor grid view and more
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-0-0/index.md:14]:
```suggestion
1. VS Code is now the core editor in CodeSandbox, enabling keybindings, user
   snippets, commands, editor grid view and more
```

File: packages/homepage/content/changelog/3-0-0/index.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+2. We now support some VS Code extensions, including native TypeScript type checking, automatic 
+import of variables, refactoring support, and VIM mode
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-0-0/index.md:16]:
```suggestion
2. We now support some VS Code extensions, including native TypeScript type
   checking, automatic import of variables, refactoring support, and VIM mode
```

File: packages/homepage/content/changelog/3-0-0/index.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+3. Added a devtool view with tabs for tests and console
+4. A new design, including menu and status bar and UI changes
+5. Added GraphQL, styled-components and yarn.lock syntax highlighting/autocompletions
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-0-0/index.md:19]:
```suggestion
5. Added GraphQL, styled-components and yarn.lock syntax
   highlighting/autocompletions
```

File: packages/homepage/content/changelog/3-3-0/index.md [hunk 16]
```diff
@@ -0,0 +1,22 @@
+---
+banner:
+title: v3.3.0
+slug: 3-3-0
+date: 2019-03-21
+description: CodeSandbox containers out of beta and improvements
+---
+
+### Containers out of Beta
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-3-0/index.md:9]:
```suggestion
### Containers out of Beta

```

File: packages/homepage/content/changelog/3-3-0/index.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+CodeSandbox Containers allow you to run almost anything you want on CodeSandbox, just like a 
+local development environment.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-3-0/index.md:11]:
```suggestion
CodeSandbox Containers allow you to run almost anything you want on CodeSandbox,
just like a local development environment.
```

File: packages/homepage/content/changelog/3-3-0/index.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+
+#### Features
+
+1. Stable support for CodeSandbox Containers, which execute the code you write on a server instead of in the browser.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-3-0/index.md:15]:
```suggestion
1. Stable support for CodeSandbox Containers, which execute the code you write
   on a server instead of in the browser.
```

File: packages/homepage/content/changelog/3-4-0/index.md [hunk 19]
```diff
@@ -0,0 +1,23 @@
+---
+banner:
+title: v3.4.0
+slug: 3-4-0
+date: 2019-12-04
+description: Say hello to Template Universe and goodbye to project setup
+---
+
+### Introducing Template Universe
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-4-0/index.md:9]:
```suggestion
### Introducing Template Universe

```

File: packages/homepage/content/changelog/3-4-0/index.md [hunk 20]
```diff
@@ -0,0 +1,23 @@
...
+Template Universe enables you to share and use templates created by the community and makes 
+it possible for the maintainers of frameworks and libraries to add official ones too.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-4-0/index.md:11]:
```suggestion
Template Universe enables you to share and use templates created by the
community and makes it possible for the maintainers of frameworks and libraries
to add official ones too.
```

File: packages/homepage/content/changelog/3-4-0/index.md [hunk 21]
```diff
@@ -0,0 +1,23 @@
...
+
+
+#### Features
+
+1. New 'Explore Templates' section added to Create Sandbox screen curating community-created 
+and official templates into categories
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-4-0/index.md:17]:
```suggestion
1. New 'Explore Templates' section added to Create Sandbox screen curating
   community-created and official templates into categories
```

File: packages/homepage/content/changelog/2-5-0/index.md [hunk 22]
```diff
@@ -0,0 +1,18 @@
...
+slug: 2-5-0
+date: 2018-02-07
+description: An editor redesign and major new features
+---
+
+### 
+
+#### Features
+
+1. Thanks to [@xyc](https://github.com/xyc), the console now supports multiline commands and syntax highlighting
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/2-5-0/index.md:13]:
```suggestion
1. The console now supports multiline commands and syntax highlighting
   ([@xyc](https://github.com/xyc))
```

File: packages/homepage/content/changelog/2-5-0/index.md [hunk 23]
```diff
@@ -0,0 +1,18 @@
...
+2. We now support configuration files for Prettier, Babel, CodeSandbox and others
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/2-5-0/index.md:14]:
```suggestion
2. We now support configuration files for Prettier, Babel, CodeSandbox and
   others
```

File: packages/homepage/content/changelog/2-5-0/index.md [hunk 24]
```diff
@@ -0,0 +1,18 @@
...
+3. Added a UI for some configuration files, like sandbox.config.json, to make changing settings more intuitive
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/2-5-0/index.md:15]:
```suggestion
3. Added a UI for some configuration files, like sandbox.config.json, to make
   changing settings more intuitive
```

File: packages/homepage/content/changelog/2-5-0/index.md [hunk 25]
```diff
@@ -0,0 +1,18 @@
...
+4. Community and official Babel plugins are now supported
+5. [@gautamarora](https://github.com/gautamarora) added native Jest integration for running tests within the editor
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/2-5-0/index.md:17]:
```suggestion
5. Added native Jest integration for running tests within the editor
   ([@gautamarora](https://github.com/gautamarora))
```

File: packages/homepage/content/changelog/2-0-0/index.md [hunk 26]
```diff
@@ -0,0 +1,14 @@
...
+slug: 2-0-0
+date: 2017-11-16
+description: Integrating with GitHub
+---
+
+### 
+
+#### Features
+
+1. You can now commit and open PRs on GitHub as well as create repositories from a sandbox
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/2-0-0/index.md:13]:
```suggestion
1. You can now commit and open PRs on GitHub as well as create repositories from
   a sandbox
```

File: packages/homepage/content/changelog/2-0-0/index.md [hunk 27]
```diff
@@ -0,0 +1,14 @@
...
+2. We've added support for static file hosting so you can use binary assets within your projects
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/2-0-0/index.md:14]:
```suggestion
2. We've added support for static file hosting so you can use binary assets
   within your projects
```

File: packages/homepage/content/changelog/1-5-0/index.md [hunk 28]
```diff
@@ -0,0 +1,24 @@
+---
+banner:
+title: v1.5.0
+slug: 1-5-0
+date: 2017-08-16
+description: A new editor and offline support
+---
+
+###
```
Comment (CompuIves) [inline packages/homepage/content/changelog/1-5-0/index.md:9]: What is this header on the top for? Not sure if we can remove this 🤔

**Comment by MichaelDeBoey** [comment]
I was thinking about reformatting the changelog list items
```md
- Sandboxes can now consume packages that use an npm alias (#3730 by @jamime)
```
That way, you have a consistent way of linking to the PR and the contributor

Thoughts about this? 🤔

File: packages/homepage/content/changelog/1-5-0/index.md (hunk 28 above)
Comment (garethx) [reply packages/homepage/content/changelog/1-5-0/index.md:9]: consistent spacing with those that do have titles

**Comment by garethx** [comment]
I prefer the cleaner look we have now. The link acknowledges the contribution, I don't think we need two links for that.
